number/src/baby_bear.rs

```diff
@@ -1,9 +1,27 @@
 use p3_baby_bear::BabyBear;
 
 use crate::powdr_field_plonky3;
+#[cfg(feature = "openvm")]
+use crate::traits::OpenVmField;
 
 powdr_field_plonky3!(BabyBearField, BabyBear);
 
+#[cfg(feature = "openvm")]
+impl OpenVmField for BabyBearField {
```

Comment:
Does this need to be in the number crate? It would make sense in the `openvm` crate, no? You should be able to implement a trait for any type type in the same trait that it is defined.